### PR TITLE
fix create genesis tool using new vm

### PR DESCRIPTION
**Pull request by ouwenkg**

### CODE REVIEW

File: tools/create-genesis/Cargo.toml [hunk 1]
```diff
@@ -9,14 +9,14 @@ serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
 json = "0.11.13"
 serde_json = "1.0.39"
-ethabi = "7.0.0"
-ethereum-types = "0.5.2"
+ethabi = "6.1.0"
+ethereum-types = "0.4"
```
Comment (leeyr338) [inline tools/create-genesis/Cargo.toml]: why ethereum types should be `0.4` ? and ethabi should be `6.2.0`

File: tools/create-genesis/src/genesis.rs [hunk 2]
```diff
@@ -1,17 +1,19 @@
+use std::collections::BTreeMap;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+use std::str::FromStr;
+
 use crate::common::{clean_0x, string_2_bytes};
```
Comment (leeyr338) [inline tools/create-genesis/src/genesis.rs]: we have a `clean_0x` function in cita_types, why not use it?

File: tools/create-genesis/src/genesis.rs [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 use crate::contracts::ContractsData;
 use crate::miner::Miner;
 use crate::params::InitData;
 use crate::solc::Solc;
-use cita_types::U256;
+
 use ethabi::Contract;
+use ethereum_types::U256;
```
Comment (leeyr338) [inline tools/create-genesis/src/genesis.rs]: Can use the type `U256` in cita_types?

Comment (ouwenkg) [reply tools/create-genesis/src/genesis.rs]: Done

File: tools/create-genesis/src/genesis.rs (hunk 2 above)
Comment (ouwenkg) [reply tools/create-genesis/src/genesis.rs]: Done

File: tools/create-genesis/Cargo.toml (hunk 1 above)
Comment (ouwenkg) [reply tools/create-genesis/Cargo.toml]: Done

File: tools/create-genesis/src/common.rs [hunk 4]
```diff
@@ -1,51 +1,15 @@
-// use ethereum_types::Public;
-pub fn clean_0x(s: &str) -> &str {
-    if s.starts_with("0x") {
-        &s[2..]
-    } else {
-        s
-    }
-}
+// pub fn clean_0x(s: &str) -> &str {
```
Comment (kaikai1024) [inline tools/create-genesis/src/common.rs]: No need to keep these codes.

File: tools/create-genesis/Cargo.toml (hunk 1 above)
Comment (ouwenkg) [reply tools/create-genesis/Cargo.toml]: ethabi 7.0.0 -> ethereum-types 0.5.2
ethabi 6.1.0 -> ethereum-types 0.4

So i dump the ethabi version.

**Pull request merged**